site/dev/common.sh

```diff
@@ -233,6 +233,16 @@ pull_versioned_docs () {
   create_nightly  
 }
 
+scan_markdown_files () {
+  echo " --> check markdown file styles"
+  pymarkdown --config markdownlint.yml scan docs/docs/nightly/docs/*.md docs/*.md README.md
+}
+
+fix_markdown_files () {
+  echo " --> fix markdown file styles"
+  pymarkdown --config markdownlint.yml fix docs/docs/nightly/docs/*.md docs/*.md README.md
```

Comment:
btw looks like there are some md files not caught by this. Some of these are "orphaned" files. 

> INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
             - about.md
             - benchmarks.md
             - fileio.md
             - security.md

Maybe we can use path expansion here to make sure we got all the md files. 

`docs/docs/**/*.md`